### PR TITLE
Feature/henry/test dandelion api

### PR DESCRIPTION
**Pull request by achaccha**
I extract all Wikipedia data related in Tech.
I apply the origin format like RssCrawler and GithubCrawler.
The Extractor folder is my test folder, so I will remove it at the next commit.
We'll refine data after this commit
Please take a look!

### CODE REVIEW

File: crawler/wiki.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+                        for j in i.attrs:
+                            if i.attrs[j][0] == "resultnum":
+                                wiki_id = i.string
+
+                        extract_entity[wiki_id] = wiki_title
+
+        return extract_entity
+
+
```
Comment (ethkim) [inline crawler/wiki.py:63]: for result.json file below, we shouldn't upload that file into github. The content is owned by wikipedia, and we shouldn't copy & redistribute the data on github. Please remove that file from the PR.

File: extractor/test_dandelion.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# SAMPLE CODE FOR DANDELION API
+# API token from @achaccha
+
+import requests
+import json
+ 
+TOKEN = '0af76af32cb343a19f7c7ea0e153922e'
```
Comment (ethkim) [inline extractor/test_dandelion.py]: Is this API token? Please do not store this on a file (it's a password for you), but instead use environmental variable to set this value at code execution time. Here's one example package that allows you to do this:

https://github.com/theskumar/python-dotenv

File: extractor/test_wikipedia.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import requests
+import json
+import ast
+from bs4 import BeautifulSoup, Tag, NavigableString   # XML(HTML)을 파싱
```
Comment (ethkim) [inline extractor/test_wikipedia.py:4]: Put all comments in English. :)

**Review by ethkim — APPROVED** [review]
Thanks!! Please see comments below, but other than that, LGTM!

File: extractor/test_dandelion.py (hunk 2 above)
Comment (harrydrippin) [reply extractor/test_dandelion.py]: I already integrated `python-dotenv` to our project. So just make sure your `.env` exists.

Comment (achaccha) [reply extractor/test_dandelion.py]: I'll make our `.env` file. 
Thank you for advice.

File: extractor/test_wikipedia.py (hunk 3 above)
Comment (achaccha) [reply extractor/test_wikipedia.py:4]: Hahaha..... My mistake... :)
I'll fix it.

File: crawler/wiki.py (hunk 1 above)
Comment (achaccha) [reply crawler/wiki.py:63]: I'll remove 'result.json' file from the PR.

**Pull request merged**